index.bs

```diff
@@ -12995,207 +12872,301 @@ with the following characteristics:
         The [=map size getter=] is a [=built-in function object=]
         whose behavior when invoked is as follows:
 
-        1.  Let |O| be the <emu-val>this</emu-val> value.
-        1.  If |O| [=is a platform object=],
-            then [=perform a security check=], passing:
-            *   the platform object |O|,
-            *   the identifier "<code>size</code>", and
-            *   the type "<code>getter</code>".
-        1.  If |O| does not [=implement=] <var ignore>A</var>, then [=ECMAScript/throw=] a {{ECMAScript/TypeError}}.
-        1.  Let |map| be the {{ECMAScript/Map}} object that is the value of |O|’s \[[BackingMap]] [=internal slot=].
-        1.  Return [=!=] <a abstract-op>Get</a>(|map|, "<code>size</code>").
+        1.  Let |O| be the <emu-val>this</emu-val> value, [=implementation-checked=] against <var ignore>a</var> with identifier "<code>size</code>" and type "<code>getter</code>".
+        1.  Let |map| be the [=map entries=] of the IDL value
+            that represents a reference to |O|.
+        1.  Return |map|'s [=map/size=], [=converted to an ECMAScript value=].
     </div>
```

Comment:
There appears to be a stray `<div>` here. If we put a `<div algorithm>` around the entire section then we could remove the `<var ignore>`.... or we could just delete this.

Reply:
Unsure what you mean here - this is the same pattern used for the algos on all the methods.

Reply:
This was bogus; I could not find the opening `<div>` for this `</div>` but it's because GitHub diffs are broken for large files.